bliss/metrics.py

```diff
@@ -43,10 +43,19 @@ class BlissMetrics(Metric):
     star_tp: Tensor
     star_fp: Tensor
 
-    # classification metrics
-    star_flux: List
-    disk_flux: List
-    bulge_flux: List
+    # Classification metrics
+    disk_frac: List
+    bulge_frac: List
+    u_flux_s: List
+    g_flux_s: List
+    r_flux_s: List
+    i_flux_s: List
+    z_flux_s: List
+    u_flux_g: List
+    g_flux_g: List
+    r_flux_g: List
+    i_flux_g: List
+    z_flux_g: List
```

Comment:
Can we store the fluxes in a tensor? That will help us to generalize the code to work for other surveys, which may have a different number of bands.

Reply:
Sounds good.

Reply:
I think we still need a list since we want to compute the median error over a variable number of sources per tile, so a fixed-size tensor wouldn't be appropriate. But we can just have it be a single list which stores all five bands, instead of one list per band.

Additional context: torchmetrics.Metric requires state to be either a tensor, or a list of tensors. So before, `star_flux` was a list of `n_d x 1` tensors, where `d = total # of sources in a batch`. Now, we can have it be a list of `n_d x 5` tensors instead, and compute the median over each band separately in `compute()`.

Reply:
I'll hold off on this for now. I like the idea of having a list of `n_d x 5` tensors instead - it should look more clean.